src/convertToObject.js

```diff
@@ -5,8 +5,19 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+
+function convertToObject(stylesString) {
+  const stylesObject = {};
+
+  stylesString.split(';').forEach((style) => {
```

Comment:
The use of `forEach` to iterate over the array is acceptable, but consider using `reduce` to build the stylesObject directly from the array, which can be more declarative.